docs/src/reference/types.md

```diff
@@ -174,6 +174,145 @@ Produces the negative of the color.
 
 - returns: color
 
+# Datetime
+
+Represents a datetime. Can be created by either specifying a custom datetime
```

Comment:
I also write these kinds of comments all the time, but "Represents a datetime" is not really helpful. ;)

Maybe something like "Represents a date, a time, or a combination of both."?